util/supported-dependency-mapping.txt

```diff
@@ -0,0 +1,6 @@
+datetime-fortran libaccessom2 cice5 mom5
+oasis3-mct libaccessom2 cice5 mom5
+libaccessom2 cice5 mom5
+cice5
+mom5
+access-om3 
```

Comment:
Is this generated from the full dependency mapping? Do we need instructions on what goes in here and how it is updated?

Reply:
Yes, it is generated from the full dependency mapping. Essentially it was just removing the models that we don't support at the moment (access-om2 among others). I will add a readme to the util folder, though,